helper/diffPlaces.js

```diff
@@ -124,20 +124,69 @@ function isNameDifferent(item1, item2, requestLanguage){
   // iterate over all the languages in item2, comparing them to the
   // 'default' name of item1 and also against the language requested by the user.
   for( let lang in names2 ){
+    if (!names2.hasOwnProperty(lang)) { continue; } // do not iterate over inherited properties
```

Comment:
these `hasOwnProperty` checks snuck in to the PR, they are not strictly related but considered good practice